content/en/docs/reference/node/systemd-watchdog.md

```diff
@@ -0,0 +1,65 @@
+---
+content_type: "reference"
+title: Kubelet Systemd Watchdog
+weight: 80
+---
+
+{{< feature-state feature_gate_name="SystemdWatchdog" >}}
+
+Using the watchdog can improve the reliability of the kubelet.
+It can be used as an alternative to periodically requesting
+the kubelet's `/healthz` endpoint for health checks. If the kubelet
+does not respond to the watchdog within the timeout period, the watchdog
+will kill the kubelet.
+
+The systemd watchdog works by requiring the service to periodically send
+a "keep-alive" signal to the systemd daemon. If the signal is not received
```

Comment:
```suggestion
a _keep-alive_ signal to the systemd process. If the signal is not received
```
Although intuition might make you think otherwise, systemd is not really a daemon.